### ISSUE TITLE
Table header "scope" attribute is incorrect for 'header: both' option

### ISSUE THREAD

**Issue by Dumluregn** [ckeditor/ckeditor4#2996]
## Type of report

Bug

## Provide detailed reproduction steps (if any)

1. Go to [https://ckeditor.com/docs/ckeditor4/latest/examples/standardpreset.html](https://ckeditor.com/docs/ckeditor4/latest/examples/standardpreset.html).
2. Create 2x2 table with `both` headers option.
3. Check the source code of table.

### Expected result

Table source code should be:
```
<table border="1" cellpadding="1" cellspacing="1" style="width:500px">
	<thead>
		<tr>
			<th scope="col">&nbsp;</th>
			<th scope="col">&nbsp;</th>
		</tr>
	</thead>
	<tbody>
		<tr>
			<th scope="row">&nbsp;</th>
			<td>&nbsp;</td>
		</tr>
	</tbody>
</table>
```

### Actual result

`scope` attribute is incorrect for `[ 1, 1 ]` cell:
```
<table border="1" cellpadding="1" cellspacing="1" style="width:500px">
	<thead>
		<tr>
			<th scope="row">&nbsp;</th>
			<th scope="col">&nbsp;</th>
		</tr>
	</thead>
	<tbody>
		<tr>
			<th scope="row">&nbsp;</th>
			<td>&nbsp;</td>
		</tr>
	</tbody>
</table>
```

## Other details

* Browser: every, e.g. Chrome v73.0.x
* OS: macOS 10.14.3
* CKEditor version: 4.11.3
* Installed CKEditor plugins: `table`

**Comment by Dumluregn** [comment]
Related to #2881 and #2882.

**Comment by bhdzllr** [comment]
I created a quick fix for this problem as a plugin by overwriting the "onOk" function for the table dialog. The modified parts are marked with "// Modified" in the code: https://gist.github.com/bhdzllr/995d71c2ee725f1cc0565d8f4dabe6c9

I just tested it by myself and did not run any automated tests with this solution.

**Comment by Dumluregn** [comment]
Thank you for taking your time to solve this issue, feel free to create PR to our repo.

**Comment by KarolDawidziuk** [comment]
Closed in #5357.
